package.json

```diff
@@ -35,8 +35,10 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
+    "@types/got": "^9.6.12",
```

Comment:
If I'm not mistaken, this should be a devDependency

Reply:
Absolutely right, a mishap on my behalf! I have moved it.